Servers/UI/OJS.Servers.Ui/ClientApp/src/layout/search-bar/SearchBar.tsx

```diff
@@ -38,16 +38,28 @@ const SearchBar = () => {
     const [ searchParam, setSearchParam ] = useState<string>(defaultState.state.searchValue);
     const [ selectedTerms, setSelectedTerms ] = useState(defaultState.state.selectedTerms);
 
-    const { state: { isVisible }, actions: { toggleVisibility } } = useSearch();
+    const { state: { isVisible, getSearchResultsUrlParams }, actions: { toggleVisibility } } = useSearch();
     const navigate = useNavigate();
+    const location = useLocation();
 
     const handleOnChangeUpdateSearch = useCallback(
         (searchInput?: IFormControlOnChangeValueType | ChangeEvent<HTMLInputElement>) => {
             setSearchParam(searchInput as string);
         },
-        [],
+        [ ],
     );
 
+    useEffect(() => {
+        if ((searchParam === defaultState.state.searchValue && selectedTerms === defaultState.state.selectedTerms) &&
+            location.pathname === '/search' && getSearchResultsUrlParams) {
```

Comment:
Try avoiding using magic strings. Create a const variable or enum and use it in that way.